modules/nw-ingress-controller-configuration-gcp-global-access.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-controller-configuration-gcp-global-access_{context}"]
+= Configuring Global Access parameters on GCP
+As a cluster administrator with an internal load balancer on GCP, you can specify the global access option. Clients in any region within the same VPC network as the load balancer can reach the workloads running on your cluster.
+
+For more information, please reference the GCP documentation for link:https://cloud.google.com/kubernetes-engine/docs/how-to/internal-load-balancing#global_access[Global access].
+
+.Prerequisites
+
+* You deployed an {product-title} cluster on GCP infrastructure.
+* You created an ingress controller with internal load balancing.
+* You installed the `oc` command line interface.
+
+.Procedure
+
+. Configure the 'clientAccess' field by specifying the appropriate level for 'clientAccess', global and local.
++
+.. Edit the Ingress Controller Operator:
++
+[source,terminal]
+----
+$ oc edit ingresses.config/cluster -o yaml
```

Comment:
An existing ingress controller can be edited such that the GCP Global Access option is enabled.
However, a new ingress controller with the option specified could be created as well. I think it may be worth mentioning this in the docs.

Reply:
@sgreene570 would it be beneficial to provide the procedure for editing an existing Ingress Controller and creating a new one?

Reply:
Note that as is, the edit instructions are based off of the wrong resource. `ingress.config` and Ingress Controllers are 2 different resources.

Reply:
Also, in response to your commend about specifying the option when you create a new ingress controller, I added a note about that in the prerequisites

Reply:
No worries, I would agree it looks like you are on the right track! Thanks!